eplmp-server/eplmp-server-ejb/src/main/java/org/polarsys/eplmp/server/ProductManagerBean.java

```diff
@@ -410,7 +413,7 @@ public PartRevision undoCheckOutPart(PartRevisionKey pPartRPK) throws NotAllowed
         User user = userManager.checkWorkspaceReadAccess(pPartRPK.getPartMaster().getWorkspace());
         Locale userLocale = user.getLocale();
 
-        PartRevision partR = partRevisionDAO.loadPartR(userLocale, pPartRPK);
+        PartRevision partR = loadRevision(userLocale, pPartRPK);
```

Comment:
Yes I saw that. Actually switching to injectable DAO is a bigger change than what we expected.
I think we should rethink about it as a whole instead of doing a little oblige change after another.